_includes/manuals/1.14/4.4.4_prov_ptables.md

```diff
@@ -3,6 +3,142 @@ Partition templates are a subset of normal Provisioning Templates. They are hand
+
+Example Dynamic Partition table:
+```
+#Dynamic - this line tells Foreman this is a script rather then a static layout
+#This snippets define the swap partition size, it would generate a partition twice the size of the memory if your physical memory is up to 2GB
+#or will create a swap partition with your memory size + 2GB.
+
+#get the actual memory installed on the system and divide by 1024 to get it in MB
+act_mem=$((`grep MemTotal: /proc/meminfo | sed 's/^MemTotal: *//'|sed 's/ .*//'` / 1024))
+
+#check if the memory is less than 2GB then swap is double the memory else it is memory plus 2 GB
+if [ "$act_mem" -gt 2048 ]; then
+    vir_mem=$(($act_mem + 2048))
+else
+    vir_mem=$(($act_mem * 2))
+fi
 
-When creating a new Host, you will be given the option to create a Dynamic Partition table. This is essentially a 'one-off' partition table that is stored with the host and used only for that host. It replaces the choice of Partition Table from the normal list of those associated with the selected OS.
+#copy all the HDD partitions to the temp file for execution
+cat <<EOF > /tmp/diskpart.cfg
+  <partitioning config:type="list">
+    <drive>
+      <device>/dev/sda</device>
+      <initialize config:type="boolean">true</initialize>
+      <partitions config:type="list">
+        <partition>
+          <create config:type="boolean">true</create>
+          <filesystem config:type="symbol">ext3</filesystem>
+          <format config:type="boolean">true</format>
+          <mount>/boot</mount>
+          <partition_id config:type="integer">131</partition_id>
+          <partition_nr config:type="integer">1</partition_nr>
+          <size>1G</size>
+          <stripes config:type="integer">1</stripes>
+          <stripesize config:type="integer">4</stripesize>
+          <subvolumes config:type="list"/>
+        </partition>
+        <partition>
+          <create config:type="boolean">true</create>
+          <format config:type="boolean">false</format>
+          <lvm_group>vg00</lvm_group>
+          <partition_id config:type="integer">142</partition_id>
+          <partition_nr config:type="integer">2</partition_nr>
+          <size>max</size>
+        </partition>
+      </partitions>
+      <pesize></pesize>
+      <type config:type="symbol">CT_DISK</type>
+      <use>all</use>
+    </drive>
+    <drive>
+      <device>/dev/vg00</device>
```

Comment:
this is the default community-templates example for an lvm host, but yeah I can make it shorter